### PR TITLE
Detail: Content margin

### PR DESCRIPTION
**Pull request by varffs**
Set margins on all responsive sizes and ensure no text and menu overlaps (except on overflow below)

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
iPad 1024 x 768 landscape

**PHOTOGRAPHE-**

screen shot 2017-11-09 at 1 26 46 pm

**Comment by mikeyrayvon** [comment]
I'm pushing a commit with a mid size between `m` and `l`: `item-ml-#`

this give us a space to have 3 columns, where **PHOTOGRAPHERS** won't fit in 4

take it or leave it.

**Comment by mikeyrayvon** [comment]
is it poss to calculate the header padding with js? based on window & content heights?

The way it is now, the content is overflowing, but it would fit if the top/bottom padding was less:

`padding: 147px 105px`
screen shot 2017-11-09 at 5 32 40 pm

`padding: 100px 105px`
screen shot 2017-11-09 at 5 34 28 pm

**Comment by mikeyrayvon** [comment]
maybe too much effort to calc padding with js...

**Comment by varffs** [comment]
how do you even get 3 columns?

**Comment by varffs** [comment]
I think that maybe we will do more complex to finish, but there will be overflow at some times so would rather we find out what the designer wants to do about that first.

**Comment by mikeyrayvon** [comment]
look at my commit

File: src/styl/site.styl [hunk 1]
```diff
@@ -153,6 +153,12 @@ font-sizes($font-size-basic, $font-size-extra)
 
 @media screen and (min-width: 1024px)
 
+  $gutter = 21px
+
+  grid-maker(680px, $gutter, ml)
```
Comment (varffs) [inline src/styl/site.styl]: if we gonna do this I think it just has to become `m` and `l` becomes `xl` etc

Comment (mikeyrayvon) [reply src/styl/site.styl]: ok i guess thats fine here cuase we dont have so much markup

Comment (mikeyrayvon) [reply src/styl/site.styl]: idk i think its better how it is, with `ml`.  thats less confusing to me somehow

**Pull request merged**